source/routing/raptor_api.cpp

```diff
@@ -1363,6 +1370,18 @@ void make_response(navitia::PbCreator& pb_creator,
         return;
     }
 
+    for (const auto stop_point_iter : *departures) {
```

Comment:
You might not want to iterate through all departures and destinations if you log information above `_TRACE` level.

It would be more interesting to log the whole collections of objects like :
```
  LOG4CPLUS_TRACE(logger, "# departure : " << *departures );
  LOG4CPLUS_TRACE(logger, "# destination : " << *destinations );
```
And you can create the appropriate `ostream& operator<<(...)` function that goes with departures and destinations.